logging/log_test.go

```diff
@@ -0,0 +1,52 @@
+package logging
+
+import (
+	"bytes"
+	log "github.com/Sirupsen/logrus"
+	"net/http"
+	"strconv"
+	"strings"
+	"testing"
+)
+
+func TestCustomOutputForApplicationLog(t *testing.T) {
+	var buf bytes.Buffer
+	Init(Options{ApplicationLogOutput: &buf})
+	msg := "Hello, world!"
+	log.Infof(msg)
+	if !strings.Contains(buf.String(), msg) {
+		t.Error("failed to use custom output")
+	}
+}
+
+func TestCustomPrefixForApplicationLog(t *testing.T) {
```

Comment:
This test validates, that the output contains the prefix. But it does not validate, that the output actually contains the full message. Seems like a possible hideout for a nice bug.